zcash_client_backend/src/fees.rs

```diff
@@ -19,6 +19,7 @@ pub mod common;
 #[deprecated(
     note = "Using this fixed fee calculation violates ZIP 317, and transactions built with it will likely fail. Use `zip317` instead."
 )]
```

Comment:
Should this remain deprecated, or should it just be behind the feature flag?

Reply:
I don't have a strong opinion on this.

Reply:
I think that just having it flagged is fine; having it both flagged and deprecated seems a bit excessive (and the deprecation causes warnings related to the tests that I can't seem to silence.)